pageComponents/NewPostPage/HeroPositionSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import Select, { Option } from "@@/components/Select";
+import { POSITION_CORRESPONDENCE_EACH_PLAYER_LENGTH } from "@@/utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  playerLength: number;
+  onChange?: (e: React.ChangeEvent<HTMLSelectElement>, value: string) => void;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function HeroPositionSelect({
+  playerLength,
+  onChange,
+  ...props
+}: Props) {
+  const [positions, setPositions] = React.useState(
+    POSITION_CORRESPONDENCE_EACH_PLAYER_LENGTH.get(2)
```

Comment:
Maybe you meant this?

```suggestion
    POSITION_CORRESPONDENCE_EACH_PLAYER_LENGTH.get(playerLength)
```